code/game/jobs/job/blackshield.dm

```diff
@@ -224,3 +225,55 @@
 	name = "Blackshield Trooper"
 	icon_state = "player-blue"
 	join_tag = /datum/job/trooper
+
+
+/datum/job/cadet
+	title = "Blackshield Cadet"
+	flag = BSCADET
+	department = DEPARTMENT_BLACKSHIELD
+	department_flag = BLACKSHIELD
+	faction = MAP_FACTION
+	total_positions = 2
+	spawn_positions = 2
+	supervisors = "the Blackshield Commander"
+	difficulty = "Easy."
+	selection_color = "#a7bbc6"
+	wage = WAGE_LABOUR_INTERN
+	health_modifier = 10
+	disallow_species = list(FORM_SOTSYNTH, FORM_AGSYNTH, FORM_NASHEF)
+
+	outfit_type = /decl/hierarchy/outfit/job/blackshield/troop/cadet
+
+	perks = list(PERK_BLACKSHIELD_CONDITIONING, PERK_BOLT_REFLECT, PERK_CHEM_CONTRABAND)
```

Comment:
```suggestion
	perks = list(PERK_BLACKSHIELD_CONDITIONING, PERK_CHEM_CONTRABAND)
```

Cadet role hasn't learned that yet, partly to discourage it being used as an extra combatant (since the total BS roles is increasing by one we want there to be some disadvantage to some of those roles), and partly because cadets might also go on to be Corpsmen, which don't get it